cmp/filter/impl/FilterChooserFieldSpec.js

```diff
@@ -154,11 +154,26 @@ export class FilterChooserFieldSpec {
     //------------------------
     // Implementation
     //------------------------
+    parseValues(values, storeRecords) {
+        return values ?? (
+            this.suggestValues ? (
+                this.fieldType === FieldType.BOOL ?
+                    [true, false] :
+                    this.extractValuesFromRecords(storeRecords)
+            ) : null
+        );
+    }
+
     parseOperators(operators) {
-        operators = operators ?? this.filterType === 'value' ? ['=', '!=', 'like'] : ['>', '>=', '<', '<=', '=', '!='];
+        operators = operators ?? this.getDefaultOperators();
         return operators.filter(it => FieldFilter.isValidOperator(it));
     }
 
+    getDefaultOperators() {
+        if (this.fieldType === FieldType.BOOL) return ['=', '!='];
```

Comment:
Supporting e.g. `!= true` would allow a user to filter on records with false OR null values in the field, but I wonder if that's something that users would expect.  As opposed to only providing `=` along with the true/false value suggestions.  

Or to put it another way - should we drop `!=` to simplify or is that functionality useful (or does it seem otherwise more consistent with both operators)?